CHANGELOG.md

```diff
@@ -38,6 +38,26 @@ imageElement.animationOptions = {
 
 ## Widget Updates
 
+### LayerList, BasemapLayerList
+
+Have you ever wanted the look and feel of the MapViewer LayerList or BasemapLayerList in your app?
+We've added a new `style` property to the LayerList and BasemapLayerList widgets to do just that.
+The `style` property currently has two possible values, `classic` and `default.`
+The `classic` style represents the user interface as it has been in previous releases.
+The `default` style brings the styling of the LayerList in MapViewer into the JavaScript SDK.
```

Comment:
Hmm I'd leave MapViewer out of this and just explain what the changes to the style actually are?

Reply:
Yeah, I agree that would be better.